src/transformers/generation_utils.py

```diff
@@ -789,6 +794,9 @@ def generate(
                 needs to be the target language token.
             forced_eos_token_id (:obj:`int`, `optional`):
                 The id of the token to force as the last generated token when :obj:`max_length` is reached.
+            save_mode (:obj:`bool`, `optional`):
+                Whether to remove possible `nan` and `inf` outputs of the model to prevent the generation method to
+                crash. Note that using ``save_mode`` can slow down generation.
 
```

Comment:
Why `save_mode`? I don't see how it's linked to `nan` and `inf`.
Also does it default to `None` because it has different defaults depending on something else or because there are three different modes (None, True, False)? Could you elaborate on the doc what happens if it's unset/set to True, set to False?

Reply:
Changed it so that it defaults back to the parameter as defined in the config. Also agree that `save_mode` is not a good name -> changed it to `remove_invalid_values` as suggested by @LysandreJik